packages/reactivity/src/effect.ts

```diff
@@ -2,7 +2,7 @@ import { OperationTypes } from './operations'
 import { Dep, targetMap } from './reactive'
 import { EMPTY_OBJ, extend } from '@vue/shared'
 
-export interface ReactiveEffect {
+export interface ReactiveEffect extends Function {
```

Comment:
```suggestion
export interface ReactiveEffect {
```

Let's keep the interface simple. (Just a suggestion, Evan WDYT?).

Reply:
The ReactiveEffect actually is the sub-class of function. So add this is better i think :)

Reply:
It is already expressed via `(): any` in the interface so I don't think the extension is necessary.